src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -162,14 +160,6 @@ internal static string GitCommitId
             }
         }
 
-        internal static Version CLRVersion
```

Comment:
Removed it since we removed the other members anyways.  After beta.3 goes out, I intend to revisit the discussion about removing FullCLR code.